overview/partner_content_reqs.md

```diff
@@ -5,4 +5,131 @@ layout: default
 section_title: Overview
 ---
 
-TODO
\ No newline at end of file
+<h2>Submitting Information for a Marketplace Listing</h2>
+
+<p>We want to help you succeed, so we created this guide to ensure we have all the necessary content to properly publish your App on the <a href ="https://marketplace.procore.com/" target="_blank">Procore Marketplace</a>. We collect the information we need from the Marketplace Listing section of your App page on the Developer Portal, and additional material you provide to our App Validation Engineer. Please ensure the content you supply for your Marketplace tile - descriptions, screenshots, video, etc. - accurately reflects the App’s core experience.</p>
+
+<p>Once you have everything assembled you can log in to your Procore Developer Portal account and visit your App page to enter and submit information for your Marketplace listing. The following sections summarize the various fields found in the Marketplace Listing section of your App page. All field entries and selections are required unless otherwise noted.</p>
+
+<h3><b>1.0 App Tab</b></h3>
```

Comment:
@jimj-procoretech  I recommend using markdown and not HTML.

Using markdown increases uniformity in terms of style and maintanability of code.
For instance `<b>` is an HTML tag that does not have an assigned style, as the tag to emphasize is `<strong>`.

Markdown also makes it easy to automatically generated ordered lists
(see the "Ordered Lists" section of the [Markdown documentation](https://guides.github.com/features/mastering-markdown/)).

By introducing HTML, we introduce complexity for the reviewers, for developers who want to collaborate on the project (e.g.: if they find a typo), and for designers to apply a consistent style to the site, without having to check that every page uses the same HTML tags.

Reply:
Can this be a fast-follow change? We've had calls into the App Validation team about this missing content. My intent with this PR to get the content up and visible as quickly as possible today.

Reply:
@jimj-procoretech I believe entering HTML in this codebase defeats the purpose of the repository itself.

I'm available to help converting this HTML file to Markdown if you are currently busy with other tasks, it should only take me a few minutes.

Reply:
Yes, that would be great. Thank you Claudio.